packages/desktop-client/src/components/accounts/Account.js

```diff
@@ -1881,13 +1952,16 @@ class AccountInternal extends PureComponent {
               fetchAllIds={this.fetchAllIds}
               registerDispatch={dispatch => (this.dispatchSelected = dispatch)}
             >
-              <View style={[styles.page, { backgroundColor: colors.n11 }]}>
+              <View style={[styles.page]}>
```

Comment:
This change makes the backdrop of the cleared & uncleared totals invisible:

<img width="419" alt="Screenshot_2023-07-02 15 45 29" src="https://github.com/actualbudget/actual/assets/25517624/e7480f50-60fd-4bfb-a74a-af4fe77ab692">

Reply:
It did the same for "applied filters" since they were set to colors.n10. Sorry I missed this one. 

Easiest fix is to change "DetailedBalance" function style from n10 to n9. I can put a PR in for it?

Reply:
PR #1265